cloud_governance/common/aws/cost_explorer/cost_explorer_operations.py

```diff
@@ -12,21 +12,22 @@ class CostExplorerOperations:
     def __init__(self):
         self.cost_explorer_client = boto3.client('ce')
 
-    def get_daily_cost_usage(self, tag: str, metrics_type: str):
+    def get_daily_cost_usage(self, tag: str, granularity: str, cost_metric: str, start_date: str = '', end_date: str = ''):
```

Comment:
I think the name should be now:
  get_cost_usage_by_granularity
and  granularity: str='DAILY', cost_metric: str='UnblendedCost'